ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -313,6 +313,33 @@ defaults <- node_group <- node_group.node <- node
     igmp_snooping_enabled: < true | false >
 ```
 
+### BGP & EVPN Multi-Domain Gateway
+```yaml
+< node_type_key >:
+
+  defaults:
+    # Node is acting as EVPN Multi-Domain Gateway | Optional.
+    # New BGP peer-group is generated between EVPN GWs in different domains or between GWs and Route Servers. Name can be changed under "bgp_peer_groups.evpn_overlay_core" variable
+    # L3 rechability for different EVPN GWs must be already in place, it is recommended to use DCI & L3 Edge if Route Servers and GWs are not defined under the same Ansible inventory.
+    evpn_gateway:
+
+      # Define remote peers of the EVPN VXLAN Gateway. If the hostname can be found in inventory, ip_address and BGP ASN will be automatically populated, unless explicitly defined, which will override the calculated parameters. If hostname can't be found, ip_address and bgp_as will be used to create the neighbor.
```

Comment:
`# Define remote peers of the EVPN VXLAN Gateway. If the hostname can be found in the inventory, ip_address and BGP ASN will be automatically populated. Manual override takes precedence. If the peer's hostname can not be found in the inventory, ip_address and bgp_as must be defined.`